### DISCUSSION TITLE
Failing site build when upgrading to version 2.0.0

### DISCUSSION THREAD

**Discussion by MaiaPelletier** [open-sdg/open-sdg#1772] [Q&A] [answered]
Site repository: https://github.com/sdg-data-canada-odd-donnees/cif-cic_dev
Failed run details: https://github.com/sdg-data-canada-odd-donnees/cif-cic_dev/runs/7752939984?check_suite_focus=true

Hello! We are trying to upgrade our data hub to version 2.0.0 and have encountered an error when pushing the changes to production. 

```
- ./_test/cif-cic_dev/search/index.html
  *  mailto:%email_address contains an invalid email address (line 164)
     <a href="mailto:%email_address">feedback</a>

HTML-Proofer found 212 failures!
Error: Process completed with exit code 1.
``` 
I'm aware of the fix https://github.com/open-sdg/open-sdg/discussions/1752 for html-proofer and have already implemented this in our Gemfile. I am just unsure why it seems to keep finding an invalid email address. Any help would be super appreciated, thank you!!

**Comment by brockfanning** [comment, marked as answer]
@MaiaPelletier In the 2.0.0 release, some of the "variables" in translations have changed. Instead of `%email_address`, we're now using something like `%email_contacts.suggestions` which corresponds to a site configuration. I would suggest first trying to go through your `translations` folder in your data repo, and change any `%email_address` to `%email_contacts.suggestions`. It will probably be in the header.yml files.

**Reply by MaiaPelletier** [reply]
Thank you!! That's what it was
